test-cloud/test-cloud-remote/src/test/filtered-resources/kjars-sources/rule-project/src/main/resources/rules.drl

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+import java.util.List;
+
+global List list
+
+rule "Fast first rule in first activation group" @Propagation(IMMEDIATE)
```

Comment:
This seems to be quite complicated rule for our purposes. I think it may be enough to have something like https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-tests/kie-server-integ-tests-drools/src/test/filtered-resources/kjars-sources/stateless-session-kjar/src/main/resources/kbase1/rules.drl